macOSUpgrade.sh

```diff
@@ -223,7 +223,10 @@ else
     freeSpace=$( /usr/sbin/diskutil info / | /usr/bin/grep "Free Space" | /usr/bin/awk '{print $6}' | /usr/bin/cut -c 2- )
 fi
 
-if [[ ${freeSpace%.*} -ge 15000000000 ]]; then
+if [[ ${freeSpace%.*} -ge 20000000000 ]] && [ "$osMajor" -gt 13 ]; then
+    spaceStatus="OK"
+    /bin/echo "Disk Check: OK - ${freeSpace%.*} Bytes Free Space Detected"
+elif [[ ${freeSpace%.*} -ge 15000000000 ]]; then
```

Comment:
I think we can make here simpler. 
Check target OS version, then set required free storage size. 
No need to use 'elif and no need take care about 15G or 20G later code at L.381.

```
if [ "$osMajor" -ge 14 ]; then 
    requiredDiskSpaceSize=20 
else
    requiredDiskSpaceSize=15
fi
if [ ${freeSpace%.*} -ge $(( requiredDiskSpaceSize * 1000 * 1000 * 1000 )) ] then
   spaceStatus=OK
  ...
else
  spaceStatus="ERROR"
 ...
fi
```

I think it is ok just "$freeSpace" instead of ${freeSpace%.*}.

Reply:
I changed this by https://github.com/kc9wwh/macOSUpgrade/pull/118/commits/617e9ea219711098e8564fe5a0360c4159424bfd.